cmd/consensus/main.go

```diff
@@ -446,8 +446,8 @@ func main() {
 			// initialize the aggregating signature module for staking signatures
 			staking := signature.NewAggregationProvider(encoding.ConsensusVoteTag, node.Me)
 
-			// initialize the threshold signature module for random beacon signatures
-			beacon := signature.NewThresholdProvider(encoding.RandomBeaconTag, privateDKGData.RandomBeaconPrivKey)
+			// initialize the verifier used to verify aggregated signatures
```

Comment:
I think the original comment is more accurate, if the module used here is about the random beacon/threshold signature.

The module declared in the line above `staking` is actually another verifier used for the aggregated signatures.